src/cfe.c

```diff
@@ -202,6 +208,9 @@ extern void cfe(
   soil_reservoir_struct->storage_m -= flux_lat_m;
   massbal_struct->vol_soil_to_lat_flow     += flux_lat_m;  //TODO add this to nash cascade as input
   massbal_struct->volout=massbal_struct->volout+flux_lat_m;
+
+  // get change in storage (used in the soil moisture coupler)
+  soil_reservoir_struct->storage_change_m = soil_reservoir_struct->storage_m - storage_temp_m ;

```

Comment:
If this is used in the soil moisture coupler, do we need it here?  Is it passed to the soil moisture coupler or SFT somewhere or is it in anticipation of coupling with the soil moisture routine?

Reply:
Yes, it is used in the soil moisture coupler and needs to be computed every time CFE is called/updated. The CFE BMI lists this variable as an output variable that will be passed to the soil moisture coupler.